envVariables.js

```diff
@@ -190,6 +190,14 @@ let enviromentVariables = {
       "value" : "OCI"
     }
   }, 
+  "KAFKA_COMMUNICATIONS_ON_OFF" : {
+    "message" : "Enable/Disable kafka communications",
+    "optional" : false
+  },
+  "KAFKA_URL" : {
+    "message" : "Required",
+    "optional" : false
+  },
```

Comment:
@VISHNUDAS-tunerlabs missing validation for PROGRAM_USERS_SUBMISSION_TOPIC, and also make use of dependent condition, PROGRAM_USERS_SUBMISSION_TOPIC and KAFKA_URL are mandatory only if KAFKA_COMMUNICATIONS_ON_OFF is ON

Reply:
@aks30 KAFKA_URL mandatory only if KAFKA_COMMUNICATIONS_ON_OFF is ON,

Reply:
@aks30 added